packages/geo/src/lib/layer/shared/layers/layer.interface.ts

```diff
@@ -63,6 +63,7 @@ export interface GeoWorkspaceQueryOptions {
 
 export interface LayersLink {
   linkId: string;
+  showInMiniBaseMap?: boolean;
```

Comment:
On devrait laisser un commentaire au développeur que la valeur par défaut est vrai et en plus, on ajoute de l'intellisense à notre type.
```suggestion
  /** Default is true */
  showInMiniBaseMap?: boolean;
```